frontend/src/static/js/static_pages/styles/AddMediaPage.scss

```
@@ -295,8 +292,10 @@ dialog {
 		@media screen and (min-width: 1024px) {
 			padding: 0.75rem 0.75rem 0;
 		}
+
 		&:hover {
 		}
```

Stylelint (17.13.0)

[error] Empty block (block-no-empty)

(block-no-empty)